Cargo.toml

```diff
@@ -27,7 +27,7 @@ maintenance = { status = "actively-developed" }
 
 [features]
 default = ["server", "client", "caching", "test-tools", "native-tls"]
-server = ["warp", "openid", "hyper", "mime", "either", "_common"]
+server = ["clap", "warp", "openid", "hyper", "mime", "either", "_common"]
```

Comment:
We should not be bringing in `clap` for a server dependency. It is fairly big and we don't want people to import it unless they are actually building the CLI.

Reply:
Removed it. Thanks!